wire-library/wire-java-generator/src/main/java/com/squareup/wire/java/JavaGenerator.java

```diff
@@ -284,7 +284,7 @@ public TypeName typeName(ProtoType protoType) {
 
   private CodeBlock singleAdapterFor(Field field, NameAllocator nameAllocator) {
     return field.getType().isMap()
-        ? CodeBlock.of("$N", nameAllocator.get(field))
+        ? CodeBlock.of("$NAdapter()", nameAllocator.get(field))
```

Comment:
Any reason for the suffix? You can name methods the same as fields.

Reply:
Doesn't it read better in the generated code? The adapter field named after its type's name is I think more confusing.